nautobot/core/tests/test_commands.py

```diff
@@ -0,0 +1,32 @@
+import collections
+from pathlib import Path
+
+from django.core.management import call_command
+import yaml
+
+from nautobot.core.testing import TestCase
+
+
+class ManagementCommandTestCase(TestCase):
+    """Test case for core management commands."""
+
+    def setUp(self):
+        """Initialize user and client."""
+        super().setUpNautobot()
+        self.user.is_superuser = True
+        self.user.save()
+        self.client.force_login(self.user)
+
+    def test_generate_performance_test_endpoints(self):
+        """Test the generate_performance_test_endpoints management command."""
+
+        call_command("generate_performance_test_endpoints")
+        endpoints_dict = yaml.safe_load(Path("./endpoints.yml").read_text())["endpoints"]
```

Comment:
Might be cleaner to redirect stdout on the `call_command()` call so that we don't clutter the disk with an `endpoints.yml` file?